src/report/graphs.py

```diff
@@ -1,40 +1,50 @@
+"""Create graphs for Trustworthy Email reports."""
+
+# Standard Python Libraries
 import math
-import numpy as np
-import matplotlib
-matplotlib.use('Agg')
-import matplotlib.pyplot as plt
-import matplotlib as mpl
-from matplotlib.patches import Rectangle, Ellipse
-from matplotlib.collections import PatchCollection
-from matplotlib.ticker import MaxNLocator
-from matplotlib.dates import DateFormatter
 from textwrap import TextWrapper
 
+# Third-Party Libraries
+import matplotlib
+
+# This line is oddly located, but it must come before the
+# matplotlib.pyplot import below.  This also creates a bunch of flake8
+# E402 errors for the imports that follow because they are not at the
+# top of the file, so I added a noqa comment for each of those lines.
+matplotlib.use("Agg")
```

Comment:
Considering all of the headaches this results in do you think it would be reasonable to use a [`matplotlibrc` file](https://matplotlib.org/2.0.2/users/customizing.html#the-matplotlibrc-file) instead? I mention it after only reading the [backend documentation](https://matplotlib.org/2.0.2/faq/usage_faq.html#what-is-a-backend) so I may not be aware of reasons why we cannot.

Reply:
This is probably possible, but I'm leery of making changes like that to this legacy code that predates me.  Also, the point of this PR is to reskeletonize this repo; I'd prefer to see this change in a separate PR.  Would you be OK with an issue?

Reply:
Yeah an issue is totally fine. I just thought it worth mentioning given the hoops being jumped through.